### PR TITLE
Select cf creds based on environment

### PR DESCRIPTION
**Pull request by jmhooper**
This commit changes the way credentials are set during CI deploys so that different environments have different credentials. It also removes the credentials from the Travis config, setting them in Travis env vars instead so they can be rotated without changing the code.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,10 +16,6 @@ before_script:
 env:
   global:
   - CF_API="https://api.fr.cloud.gov"
-  - CF_USERNAME="gsa-18f-federalist_deployer"
-  - CF_ORGANIZATION="gsa-18f-federalist"
```
Comment (brandocalrissian) [inline .travis.yml]: As long as CF_ORGANIZATION is set in the environment, too, we should be good.

Comment (jmhooper) [inline .travis.yml]: It is not. I think in my mind I was deleting `CF_PASSWORD`.

Comment (jmhooper) [inline .travis.yml]: Another good catch to start the day 🎣

**Comment by jmhooper** [comment]
Amended the commit to put the `CF_ORGANIZATION` var back.

**Pull request merged**